contrib/playlib/src/mill/playlib/RouterModule.scala

```diff
@@ -0,0 +1,120 @@
+package mill
+package playlib
+
+import coursier.{Cache, MavenRepository}
+import mill.api.Loose
+import mill.playlib.api.RouteCompilerType
+import mill.scalalib.Lib.resolveDependencies
+import mill.scalalib._
+import mill.scalalib.api._
+
+trait RouterModule extends mill.Module with ScalaModule {
+
+  /**
+    * Defines the version of playframework to be used to compile this projects
+    * routes.
+    */
+  def playVersion: T[String]
+
+  /**
+    * The scala version to use for this project. The version defaults to
+    * the value chosen by play but some version are crossbuilt for multiple
+    * versions of scala.
+    * Override it if the default doesn't match your current usage.
+    */
+  override def scalaVersion: T[String] = T {
```

Comment:
I'm not sure, I like that default here. What about other modules, that also like to override `scalaVersion`.

Reply:
The play sbt plugin provides a default scala version which is why I added this default behaviour. 
Since all modules inherit from ScalaModule the last mixed-in will "define" the actual scalaVersion used and the end user can always override in his own object. 

With that said I don't specifically care either way as I always define my own explicit scalaVersion in my app build definition.

Reply:
I think it might be a bit too early to define a default value. I would let the version "undefined" so the user must provide a version.
And we can always come back to it and define a default value if we feel like it's tedious to declare the Scala version.